css/base.css

```diff
@@ -0,0 +1,1018 @@
+::-moz-selection {
+  background: yellow;
+}
+::selection {
+  background: yellow;
+}
```

Comment:
Better to use some file with color variables to prevent long time spending for changing it. For now you can use CSS variables.